src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AndroidBrokerProxy.cs

```diff
@@ -205,16 +205,30 @@ public AdalResultWrapper GetAuthTokenInBackground(AuthenticationRequest request,
         public Intent GetIntentForBrokerActivity(AuthenticationRequest request, Activity callerActivity)
         {
             Intent intent = null;
-            IAccountManagerFuture result = null;
+            
             try
             {
+                IAccountManagerFuture result = null;
                 // Callback is not passed since it is making a blocking call to get
                 // intent. Activity needs to be launched from calling app
                 // to get the calling app's metadata if needed at BrokerActivity.
                 Bundle addAccountOptions = GetBrokerOptions(request);
+
+                _logger.Info("BrokerProxy: Broker options:" +
+                    "\nAuthority: " + request.Authority +
+                    "\nBroker Account Name: " + request.BrokerAccountName +
+                    "\nClaims: " + request.Claims);
+
                 result = _androidAccountManager.AddAccount(BrokerConstants.BrokerAccountType,
                     BrokerConstants.AuthtokenType, null, addAccountOptions, null,
-                    null, new Handler(callerActivity.MainLooper));
+                    null, GetPreferredLooper(callerActivity));
+
+                if(result == null)
+                {
+                    _logger.Info("BrokerProxy: IAccountManagerFuture is null. ");
+                }
+
+                _logger.Info("BrokerProxy: IAccountManagerFuture returned. ");
```

Comment:
>_logger.Info("BrokerProxy: IAccountManagerFuture returned. "); [](start = 16, length = 62)

this will always be called, did you want more of an else above?

Reply:
i'm going to remove this...was just to get more info on where we were at w/the customer and debugging.

---
In reply to: [286274620](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1598#discussion_r286274620) [](ancestors = 286274620)